scapy/layers/tls/handshake.py

```diff
@@ -1056,6 +1059,7 @@ class TLS13CertificateRequest(_TLSHandshake):
                    _ExtensionsField("ext", None,
                                     length_from=lambda pkt: pkt.msglen -
                                     pkt.cert_req_ctxt_len - 3)]
+
```

Comment:
Is this change needed?

Reply:
This one returned a flake8 error (i think)